guide/src/storage.md

```diff
@@ -0,0 +1,112 @@
+<!-- 
+Copyright 2025 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Using Google Cloud Storage
+
+Google [Cloud Storage] is a managed service for storing unstructured data. The
+Rust client library offer an idiomatic API to access to this service. The client
```

Comment:
s/offer/offers/ 

s/to access to this/to access this/

Reply:
Done.